iocore/net/SSLConfig.cc

```diff
@@ -439,6 +439,23 @@ SSLConfigParams::getClientSSL_CTX() const
 }
 
 void
+SSLConfigParams::LoadTicket() const
+{
+  ticket_block_free(default_global_keyblock);
```

Comment:
I likely don't understand this code... But how is this not a race condition? You delete the key block (bad name btw, since it's part of the SSLConfigParams() class), and then create a new one. But the same SSLConfigParams could still be in use by another process?

Shouldn't the entire SSLConfigParams() object be "reloaded" and recycled?